src/starter-kit wiki page folder/_Footer.md

```diff
@@ -0,0 +1,3 @@
+### Want to add to this wiki? Check out our [contributing guidelines](https://github.com/bids-standard/bids-starter-kit/blob/master/CONTRIBUTING.md)

```

Comment:
I think we can delete this file: if we need a specific footer we will get its content back

Reply:
Sure, will do.

Reply:
Already made the change. Thank you.